fieldservice/models/fsm_order.py

```diff
@@ -328,6 +328,10 @@ def action_cancel(self):
             {"stage_id": self.env.ref("fieldservice.fsm_stage_cancelled").id}
         )
 
+    @api.onchange("scheduled_date_start")
```

Comment:
@marcelsavegnago Why not adding this field in the decorator of `onchange_scheduled_duration()` directly?

Reply:
Done